documentation/API-Readiness-Checklist.md

```diff
@@ -2,26 +2,27 @@
 
 Checklist for api-name api-version in rx.y.
 
-| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |      | link |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      |      |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      |      |
-|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      |      |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |      | Comm. release nr |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |      | ICM release nr |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |      | x.y.z |
```

Comment:
x.y.z is doubling the information from line 3 here ("api-version"). And there could be other formats as `x.y.z`, e.g. `x.y.z-rc.n`
I wouldn't require here a reference information.

Reply:
indeed, will remove it